### ISSUE TITLE
Add additional alerting methods for the elastalert backend

### ISSUE THREAD

**Issue by christophetd** [Neo23x0/sigma#256]
Hello, 

I'm playing around with Sigmac and Elastalert, and there are a few additions that would be great to have for the elastalert backend.

The main one I see is being able to choose an `alert_method` to determine how elastalert will send an alert. Currently [only e-mail is supported](https://github.com/Neo23x0/sigma/blob/master/tools/sigma/backends/elasticsearch.py#L567-L572), I think adding HTTP POST webhooks would be a good first step. In this case, we'd also need to add [the relevant elastalert options](https://elastalert.readthedocs.io/en/latest/ruletypes.html#http-post), and it probably already becomes too much for passing via the CLI, so we would probably need #213 first.

Thoughts? Happy to work on a PR for that.

**Comment by andres-tw** [comment]
An option I've also been using (Will try to make a PR soon) Is to utilize the `import` keyword supported by elastalert.

With this option, you could have Sigma generate the elastalert rule and append a single line at the end like: `import: /path/to/extra/config.conf`

Then in `/path/to/extra/config.conf` you can add additional elastalert options like:
```
alert: post
http_post_url: "http://example.com/api"
http_post_payload:
  ip: clientip
http_post_static_payload:
  apikey: abc123
```
But this in its current format is a simple append contents of file, but makes it so you can add common options to multiple rules when elastalert loads them.
In the same frame I've been toying with the idea of teaching this elastalert import keyword some yaml merging capabilites, so you could have common options in the import config file and override or extend specific keys in the rule file.

**Comment by haam3r** [comment]
An option I've also been using (Will try to make a PR soon) Is to utilize the `import` keyword supported by elastalert.

With this option, you could have Sigma generate the elastalert rule and append a single line at the end like: `import: /path/to/extra/config.conf`

Then in `/path/to/extra/config.conf` you can add additional elastalert options like:
```
alert: post
http_post_url: "http://example.com/api"
http_post_payload:
  ip: clientip
http_post_static_payload:
  apikey: abc123
```
But this in its current format is a simple append contents of file, but makes it so you can add common options to multiple rules when elastalert loads them.
In the same frame I've been toying with the idea of teaching this elastalert import keyword some yaml merging capabilites, so you could have common options in the import config file and override or extend specific keys in the rule file.

**Comment by christophetd** [comment]
FYI, I actually implemented HTTP POST alerting as part of #258 (and forgot to close this issue). I'd  suggest you open a new issue to discuss your suggestion and we close this one?

**Comment by haam3r** [comment]
Sounds good to me
